addons/expanding-search-bar/expanding-search-bar.js

```diff
@@ -69,15 +89,19 @@ export default async function ({ addon, console }) {
       function exsearch_clickIn() {
         //Clicking into the search bar
         if (addon.self.disabled) return; //Don't expand if addon disabled
-        exsearch_siteNav.style.display = "none"; //Hide the site navigation
+        exsearch_siteNav.style.width = "0px"; //Hide the site navigation
```

Comment:
What's the difference between display:none and setting the width to 0px?

Reply:
> A side effect of this change is that pressing Shift+Tab now moves to the Scratch logo instead of "About" on the navbar, so the way the links are hidden has been changed so pressing Shift+Tab (to move to the previous element) when navigating with a keyboard still moves focus to "About". Keyboard usability is useful not only for accessibility but also efficiency (I could imagine people Tabbing to the search bar all the time).

Reply:
To clarify, setting "display" to "none" makes an element not keyboard-focusable, but setting "width" to "0px" keeps its keyboard-focusability.

Reply:
Maybe add a small comment to this line that says "allow keyboard focus with Tab key"?

Reply:
Done!